### ISSUE TITLE
Change icon colors away from black

### ISSUE THREAD

**Issue by bonimy** [Maseya/Brutario#64]
They're not visible in high contrast mode..
<img width="709" height="92" alt="Image" src="https://github.com/user-attachments/assets/c8174ee1-b312-4969-bbff-ca292eb3b045" />
